sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -2509,10 +2509,10 @@
               "lockKey": true
             },
             {
-              "input": true,
+              "label": "Is it impossible for you to provide your parents' financial information and you need to request consideration for modified independant status due to exceptional circumstances?",
```

Comment:
Please ensure the below is checked.
![image](https://github.com/user-attachments/assets/d9ca2544-d754-4870-9a2b-f7a64c4aaf24)

Reply:
updated.